tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs

```diff
@@ -1209,12 +1211,12 @@ public class ExpressionParserTests
             Test("convertToUTC('01/01/2018 00:00:00', 'Pacific Standard Time', 'D')", "lundi 1 janvier 2018"),
             Test("convertFromUTC('2018-01-02T02:00:00.000Z', 'Pacific Standard Time', '')", "01/01/2018 18:00:00"),
             TestUtcNow("utcNow('D')", () => DateTime.UtcNow.ToString("D", new CultureInfo("fr-FR"))),
-            TestUtcNow("getPastTime(1,'Day', 'D')", () => DateTime.Now.AddDays(-1).ToString("D", new CultureInfo("fr-FR"))),
+            TestUtcNow("getPastTime(1,'Day', 'D')", () => DateTime.UtcNow.AddDays(-1).ToString("D", new CultureInfo("fr-FR"))),
```

Comment:
I changed this to UtcNow because if you run it late enough in the day it fails due to UTC being a different day. These are explicitly testing "UtcNow" but the expected value wasn't UtcNow. I suspect it was just an oversight.